ci/containers/build.Containerfile

```diff
@@ -45,6 +44,12 @@ ENV PATH $SPACK_ROOT/bin:/root/.local/bin:$PATH
 
 RUN spack repo add --scope site $SPACK_PACKAGES_ROOT/repos/spack_repo/builtin
 
+# FIXME: Workaround until CE provides full MPI replacement
+# Add custom repo, if it exists in the base image
+RUN if [ -d /root/site/spack_repo/alps/ ]; then \
+      spack repo add --scope site /root/site/spack_repo/alps/; \
+    fi
+
```

Comment:
Note that in the latest version of https://github.com/eth-cscs/DLA-Future/pull/1311 I clone the alps-cluster-config repo directly in the container build to avoid having a custom base image (they achieve the same purpose, I just prefer avoiding the manually built base image).

Reply:
Good point, I'll try to do the same.